pkg/scheduler/ugm/manager_test.go

```diff
@@ -337,10 +325,93 @@ func TestUpdateConfig(t *testing.T) {
 	conf = createConfig(user.User, user.Groups[0], "memory", "50", 10, 10)
 	err = manager.UpdateConfig(conf.Queues[0], "root")
 	assert.NilError(t, err)
-	increased = manager.IncreaseTrackedResource(queuePath1, TestApp1, usage, user)
-	if increased {
-		t.Fatalf("unable to increase tracked resource: queuepath %s, app %s, res %v", queuePath1, TestApp1, user)
+	headroom = manager.Headroom(queuePath1, TestApp1, user)
+	assert.Equal(t, headroom.FitInMaxUndef(usage), false)
+}
+
+func TestUseWildCard(t *testing.T) {
+	setupUGM()
+	manager := GetUserManager()
+	user := security.UserGroup{User: "user1", Groups: []string{"group1"}}
+
+	expectedResource, err := resources.NewResourceFromConf(map[string]string{"memory": "50", "vcores": "50"})
+	if err != nil {
```

Comment:
use `assert.NilError()` to check for nil error returns (multiple instances found)